### PR TITLE
Adding support for .NET Standard 2.0

### PR DESCRIPTION
**Pull request by fabiocav**
This PR adds support for .NET Standard 2.0, aiming at supporting .NET Framework OOP applications. 

Adding @mattchenderson for awareness. This will require work on templates and tooling, but I'm not currently looking at that as blockers.

### Pull request checklist

* [ ] My changes **do not** require documentation changes
  * [x] Otherwise: Documentation issue linked to PR (coming. will sync with @mattchenderson )
* [ ] My changes **should not** be added to the release notes for the next release
  * [x] Otherwise: I've added my notes to `release_notes.md`
* [x] My changes **do not** need to be backported to a previous version
  * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [x] I have added all required tests (Unit tests, E2E tests)

### CODE REVIEW

**Review by brettsam — APPROVED** [review]
LGTM!

File: src/DotNetWorker.Grpc/Http/GrpcHttpRequestData.cs [hunk 1]
```diff
@@ -146,13 +155,18 @@ public void Dispose()
...
 
             List<IHttpCookie> httpCookiesList = new List<IHttpCookie>(separateCookies.Length);
 
             for (int c = 0; c < separateCookies.Length; c++)
             {
-                var splitArray = separateCookies[c].Split("=", StringSplitOptions.RemoveEmptyEntries);
+#if NET5_0_OR_GREATER
+                var separator = '=';
+#else
+                var separator = new[] { '=' };
```
Comment (kshyju) [inline src/DotNetWorker.Grpc/Http/GrpcHttpRequestData.cs:167]: Unrelated to this PR. But the separator doesn't need to be allocated for each iteration of the loop. Could be moved out.

**Review by kshyju — APPROVED** [review]

**Comment by fabiocav** [comment]
Merging. Will follow up with samples and work with @mattchenderson on next steps

**Pull request merged**